### DISCUSSION TITLE
購読しているアドベントカレンダーの通知は通常の通知とはわけてほしい

### DISCUSSION THREAD

**Discussion by munierujp** [increments/qiita-discussions#577] [要望・アイデア・不具合]
アドベントカレンダーを購読すると記事が投稿されたときに通知が来ますが、いくつかのカレンダーを購読していると毎日数十件単位の通知が届きます。
そうなると複数ページにわたって表示されるので確認するのが大変で、とりこぼす可能性もあります。
そのため、購読機能は使わずにRSSをRSSリーダーで購読しているのが現状です。
アドベントカレンダーの通知に関しては、通常の通知とはわけて管理できるようにしていただきたいです。

**Comment by tanaka-asami415** [comment]
@munierujp 
リクエストありがとうございます🙏

ご不便をおかけして申し訳ございません。
通知関連のリクエストは他にもいただいており、我々も改善したいと思っておりますので、
来年のアドベントカレンダーに向けて検討いたします🙇

**Reply by P-SiZK** [reply]
@munierujp 
お待たせしました :bow: 
通知をカテゴリーごとに確認できる機能を追加しました。
https://qiita.com/release-notes/2024-09-11

アドベントカレンダーのタブで絞り込まれた通知を以下 URL から確認できます！
https://qiita.com/notifications?scope=advent_calendar

**Reply by tanaka-asami415** [reply]
大変申し訳ありません。
ご報告したばかりなのですが、一部のユーザー様において表示が重い状況が発生しており、もしかすると表示されない事があるかもしれません。

現在修正中なので、完了したら改めてご報告いたします🙇
https://x.com/Qiita/status/1836678416395047139

**Reply by tanaka-asami415** [reply]
お待たせいたしました。修正を行い、問題なくご利用いただける状態となりました🙇

ぜひご活用いただけると嬉しいです！
https://x.com/Qiita/status/1838396877546676301
